infer_subc/organelles/cellmask.py

```diff
@@ -117,6 +123,187 @@ def choose_max_label_cellmask_union_nucleus(cellmask_img: np.ndarray,
 
     return cellmask_out > 0
 
+def rescale_intensity(in_img: np.ndarray):
+    #rescales the intensity of input image on a scale of 0 to 10
+    out_img = ((in_img - in_img.min())/(in_img.max() - in_img.min()))*10
+    return out_img
+
+def create_composite(in_img: np.ndarray,
```

Comment:
Seems like this function is better worked out in the membrane.py file. Are they the same? If so, I think it should live only in the membrane.py location.

Reply:
yeah, it works better in the membrane.py file, that other one is an artifact from an earlier push to github and is no longer present in my code, I can go ahead and make github reflect that on monday